### PR TITLE
Kiselitza add cerbos to misc

### PR DESCRIPTION
**Pull request by kiselitza**
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 250 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation
  such as title case, all caps, small caps or all lowercase.
  `Demo` links should only be used for interactive demos, i.e. not video demonstrations.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that depend on proprietary services outside the user's control must be marked `⚠`.
  ``- [Name](http://homepage/) `⚠` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software) must be added to `non-free.md` and have the license set to `⊘ Proprietary`
- [x] Additions are inserted preserving alphabetical order.
- [x] Additions are not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The `Language` tag is the main **server-side** requirement for the software. Don't include frameworks or specific dialects.
- [x] Any license you add is in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [x] Any category you are creating has the minimum requirement of 3 items.
  If not, your addition may be inserted into `Miscellaneous`.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] If you are adding software forked from another project, please provide/link to a clear list of differences between both.
- [x] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

### CODE REVIEW

**Comment by kiselitza** [comment]
Hi! Thanks for looking into it.
I did check awesome-sysadmin, and the only thing that would make sense to me is the creation of an authZ category there, but that would force me to bring up 2 more tools (3 are asked for a new category) for which I can't provide proper feedback or inputs.

I'd argue that the suggested category talks about identity management, and Cerbos doesn’t care about identity (you being your own identity provider or build it yourself -whatever). Cerbos only cares about access, which I don’t see as a part of it (if it were IAM, not just IM - then fine).
Is that something you’re strongly opinionated about, or you have some other suggestions potentially? 😄

**Comment by kiselitza** [comment]
Hi @nodiscc mate :)
Any thoughts on the above?

**Comment by nodiscc** [comment]
Hi, sorry for the late reply, I think the proper procedure is to add it to a Miscellaneous section in awesome-sysadmin (create it if it doesnt exist - disregard the minimum of 3 items rule), it could be moved to a new Authorization Management section once 2 other similar projects are submitted/reviewed.

**Comment by kiselitza** [comment]
Heey, thx!
Well, technically only 1 more is needed, I see you got this in awesome-selfhosted Miscellaneous category (where I wanted to fit Cerbos for now as well):
`[Teleport](https://goteleport.com/) - Certificate authority and access plane for SSH, Kubernetes, web applications, and databases. ([Source Code](https://github.com/gravitational/teleport)) Apache-2.0 Go`

This tool fits the access management realm like cerbos does, just on an infrastructural rather than application layer.
Why not keep them here together until there is one more tool that ticks the same box, and then move them altogether to awesome-sysadmin? 😄

**Comment by nodiscc** [comment]
> Why not keep them here together until there is one more tool that ticks the same box, and then move them altogether to awesome-sysadmin?

Sounds good.

**Comment by kiselitza** [comment]
Sweet, now waiting for some user reviews, right? 😃

File: README.md [hunk 1]
```diff
@@ -1325,6 +1325,7 @@ Software that does not fit in another section.
 - [asciinema](https://github.com/asciinema/asciinema-server) - Web app for hosting asciicasts. ([Demo](https://asciinema.org/)) `Apache-2.0` `Elixir/Docker`
 - [Baby Buddy](https://github.com/babybuddy/babybuddy) - Helps caregivers track baby sleep, feedings, diaper changes, and tummy time. ([Demo](https://demo.baby-buddy.net/login/?next=/)) `BSD-2-Clause` `Python`
 - [CapRover](https://caprover.com/) - Build your own PaaS in a few minutes. ([Demo](https://captain.server.demo.caprover.com/#/login), [Source Code](https://github.com/caprover/caprover)) `Apache-2.0` `Docker/Nodejs`
+- [Cerbos](https://cerbos.dev/) - A self-hosted, open source user authorization layer for your applications. ([Source Code](https://github.com/cerbos/cerbos)) ([Demo](https://play.cerbos.dev)) `Apache-2.0` `Go`
```
Comment (KieranRobson) [inline README.md:1328]:
```suggestion
- [Cerbos](https://cerbos.dev/) - A self-hosted, open source user authorization layer for your applications. ([Demo]((https://play.cerbos.dev), [Source Code](https://github.com/cerbos/cerbos)) `Apache-2.0` `Go`

```

**Review by KieranRobson — CHANGES_REQUESTED** [review]
Quick change I noticed.

File: README.md [hunk 2]
```diff
@@ -1325,7 +1325,8 @@ Software that does not fit in another section.
 - [asciinema](https://github.com/asciinema/asciinema-server) - Web app for hosting asciicasts. ([Demo](https://asciinema.org/)) `Apache-2.0` `Elixir/Docker`
 - [Baby Buddy](https://github.com/babybuddy/babybuddy) - Helps caregivers track baby sleep, feedings, diaper changes, and tummy time. ([Demo](https://demo.baby-buddy.net/login/?next=/)) `BSD-2-Clause` `Python`
 - [CapRover](https://caprover.com/) - Build your own PaaS in a few minutes. ([Demo](https://captain.server.demo.caprover.com/#/login), [Source Code](https://github.com/caprover/caprover)) `Apache-2.0` `Docker/Nodejs`
-- [Cerbos](https://cerbos.dev/) - A self-hosted, open source user authorization layer for your applications. ([Source Code](https://github.com/cerbos/cerbos)) ([Demo](https://play.cerbos.dev)) `Apache-2.0` `Go`
+- [Cerbos](https://cerbos.dev/) - A self-hosted, open source user authorization layer for your applications. ([Demo]((https://play.cerbos.dev), [Source Code](https://github.com/cerbos/cerbos)) `Apache-2.0` `Go`
```
Comment (KieranRobson) [inline README.md:1328]:
```suggestion
- [Cerbos](https://cerbos.dev/) - A self-hosted, open source user authorization layer for your applications. ([Demo](https://play.cerbos.dev), [Source Code](https://github.com/cerbos/cerbos)) `Apache-2.0` `Go`
```

File: README.md [hunk 3]
```diff
@@ -1325,7 +1325,8 @@ Software that does not fit in another section.
...
+
```
Comment (KieranRobson) [inline README.md:1329]: Remove this new line

**Review by KieranRobson — CHANGES_REQUESTED** [review]
I added an extra `(` by mistake, and there seems to be a new line added. Could you make these changes? Apologies for the extra `(`!

**Comment by nodiscc** [comment]
Hi,
thank you for your contribution. Today an improved version of awesome-selfhosted goes live at https://awesome-selfhosted.net/. Please check the [release announcement](https://github.com/awesome-selfhosted/awesome-selfhosted/releases/tag/1.0.0) for more details.

Contributions should now be made through https://github.com/awesome-selfhosted/awesome-selfhosted-data. If you are still interested in having this software added to the list, please check contribution guidelines there, and open a new PR/issue.

**Pull request closed without merging**